File: fixtures/scale/php/large/src/Families/Beaufort/BeaufortVectors.php
<?php

namespace CipherLab\Families\Beaufort;

use CipherLab\Abstractions\TestVector;

// Known-answer test data captured from BeaufortCipher's own encode().
class BeaufortVectors
{
    /** @return TestVector[] */
    public static function all(): array
    {
        return [
            new TestVector("ATTACKATDAWN", "JDEMPYPJUSPH"),
            new TestVector("THEQUICKBROWNFOX", "CRPCHWRASJHQIBLV"),
            new TestVector("DEFENDTHEEASTWALL", "MOQQARIXVWTMOSXJK"),
        ];
    }
}

File: fixtures/scale/csharp/large/Families/Beaufort/BeaufortVectors.cs
using System.Collections.Generic;
using CipherLab.Abstractions;

namespace CipherLab.Families.Beaufort
{
    public static class BeaufortVectors
    {
        public static List<TestVector> All()
        {
            return new List<TestVector>
            {
                new TestVector("ATTACKATDAWN", "JDEMPYPJUSPH"),
                new TestVector("THEQUICKBROWNFOX", "CRPCHWRASJHQIBLV"),
                new TestVector("DEFENDTHEEASTWALL", "MOQQARIXVWTMOSXJK"),
            };
        }
    }
}
